code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -429,26 +430,26 @@
 
 		var/atom/movable/screen/storage/background/stored_continue = new /atom/movable/screen/storage/background(null, O, "stored_continue")
 		var/matrix/M_continue = matrix()
-		M_continue.Scale((endpoint-startpoint-stored_cap_width*2)/32,1)
-		M_continue.Translate(startpoint+stored_cap_width+(endpoint-startpoint-stored_cap_width*2)/2 - 16,0)
+		M_continue.Scale(stored_width / 32,1)
+		M_continue.Translate(startpoint - 16 + ((stored_width + stored_cap_width*2) / 2),0) // -16 to center the sprite, since we're 32x32
```

Comment:
```suggestion
		M_continue.Translate(startpoint - 16 + ((stored_width + stored_cap_width*2) / 2), 0) // -16 to center the sprite, since we're 32x32
```